docker/Dockerfile

```diff
@@ -28,70 +23,30 @@ RUN echo "## Start building" \
             ca-certificates \
             build-essential \
             file \
-            python python-pip \
             python3 python3-pip \
-    \
 &&  echo "## Done"
 
 RUN  echo "## Get EMSDK" \
     &&  git clone https://github.com/emscripten-core/emsdk.git ${EMSDK} \
-    &&  cd ${EMSDK} && git reset --hard ${EMSDK_CHANGESET} \
-    \
-    &&  ./emsdk.py update-tags \
 &&  echo "## Done"
 
 RUN  echo "## Install Emscripten" \
     &&  cd ${EMSDK} \
-    && ./emsdk install ${EMSCRIPTEN_VERSION} \
-    \
+    &&  ./emsdk install ${EMSCRIPTEN_VERSION} \
 &&  echo "## Done"
 
 # This generates configuration that contains all valid paths according to installed SDK
 RUN cd ${EMSDK} \
     &&  echo "## Generate standard configuration" \
-    \
     &&  ./emsdk activate ${EMSCRIPTEN_VERSION} --embedded \
-    &&  ./emsdk construct_env > /dev/null \
+    &&  ./emsdk construct_env \
     &&  cat ${EMSDK}/emsdk_set_env.sh \
-    \
-    # remove wrongly created entry with EM_CACHE, variable will be picked up from ENV
-    &&  sed -i -e "/EM_CACHE/d" ${EMSDK}/emsdk_set_env.sh \
-    # add a link to tools like asm2wasm in a system path
-    # asm2wasm (and friends might be places either in ./upstream of ./fastcomp folder, hence detection is needed)
-    &&  printf "export PATH=$(dirname $(find . -name asm2wasm -exec readlink -f {} +)):\$PATH\n" >> ${EMSDK}/emsdk_set_env.sh \
-    \
-&&  echo "## Done"
-
-# Create a structure and make mutable folders accessible for r/w
-RUN cd ${EMSDK} \
-    &&  echo "## Create .data structure" \
-    &&  for mutable_dir in ${EM_DATA} ${EM_PORTS} ${EM_CACHE} ${EMSDK}/zips ${EMSDK}/tmp; do \
-            mkdir -p ${mutable_dir}; \
-            chmod -R 777 ${mutable_dir}; \
-        done \
-        \
+    &&  chmod -R 777 ${EMSDK} \
 &&  echo "## Done"
 
 # Create an entrypoint that activates Emscripten SDK and helps running this image as non-root user
 COPY entrypoint ${EMSDK}/
 
-
-# Create symbolic links for critical Emscripten Tools
```

Comment:
This change was fixing a problem if docker image was used on some of CI platform (CircleCI or Travis) and that platform by default was replacing entrypoint so we couldn't rely on activating emscripten prior to compilation. 

I think that the change is in general fine - but that will reduce applicability of the image.

Reply:
Oh.. I think I can replace this with just adding emscripten to the PATH.. that should do it.

Reply:
OK, I updated to rely only on PATH.. with no need to activate.

Seems better to have just a single way do things, no ?   Plus it simplifies the entry point somewhat.